praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/service/PraktikumsstellenService.java

```diff
@@ -18,12 +22,29 @@ public class PraktikumsstellenService {
     private final StudiumsPraktikumsstellenRepository studiumsPraktikumsstellenRepository;
     private final AusbildungsPraktikumsstellenRepository ausbildungsPraktikumsstellenRepository;
 
-    public StudiumsPraktikumsstelleDTO saveStudiumsPraktikumsstelle(final CreateStudiumsPraktikumsstelleDTO createStudiumsPraktikumsstelleDTO) {
+    private final MeldezeitraumRepository meldezeitraumRepository;
+
+    public StudiumsPraktikumsstelleDTO saveStudiumsPraktikumsstelle(CreateStudiumsPraktikumsstelleDTO createStudiumsPraktikumsstelleDTO) {
+        createStudiumsPraktikumsstelleDTO = createStudiumsPraktikumsstelleDTO.withId(getCurrentMeldezeitraum().getId());
         return praktikumsstellenMapper.toDTO(studiumsPraktikumsstellenRepository.save(praktikumsstellenMapper.toEntity(createStudiumsPraktikumsstelleDTO)));
     }
 
-    public AusbildungsPraktikumsstelleDTO saveAusbildungsPraktikumsstelle(final CreateAusbildungsPraktikumsstelleDTO createAusbildungsPraktikumsstelleDTO) {
+    public AusbildungsPraktikumsstelleDTO saveAusbildungsPraktikumsstelle(CreateAusbildungsPraktikumsstelleDTO createAusbildungsPraktikumsstelleDTO) {
+        createAusbildungsPraktikumsstelleDTO = createAusbildungsPraktikumsstelleDTO.withId(getCurrentMeldezeitraum().getId());
         return praktikumsstellenMapper
                 .toDTO(ausbildungsPraktikumsstellenRepository.save(praktikumsstellenMapper.toEntity(createAusbildungsPraktikumsstelleDTO)));
     }
+
+    public Meldezeitraum getCurrentMeldezeitraum() {
+        Meldezeitraum currentMeldezeitraum = null;
+        for (Meldezeitraum meldezeitraum : meldezeitraumRepository.findAll()) {
+            // plus and minus make reports the same day as start or end possible
+            if (LocalDate.now().isAfter(meldezeitraum.getStartZeitpunkt().minusDays(1)) &&
+                    LocalDate.now().isBefore(meldezeitraum.getEndZeitpunkt().plusDays(1))) {
```

Comment:
the code is better readable and does not need additional comment in two possible ways
- LocalDate.now().isAfter(meldezeitraum.getStartZeitpunkt()) || LocalDate.now().isEqual(meldezeitraum.getStartZeitpunkt())
- a method with the name isEqualOrAfter() than u could use minusDays(1)

I would prefere the first version.

The check can also be moved to the entity or the dto